### PR TITLE
SREP-1550: Add PrometheusRules for rbac-permission-operator

### PR DESCRIPTION
**Pull request by lisa**
Signed-off-by: Lisa Seelye <lseelye@redhat.com>

### CODE REVIEW

File: deploy/sre-prometheus/100-rbac-permission-operator.PrometheusRule.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-rbac-permission-operator
+    role: alert-rules
+  name: sre-rbac-permission-operator
+  namespace: openshift-monitoring
```
Comment (fahlmant) [inline deploy/sre-prometheus/100-rbac-permission-operator.PrometheusRule.yaml]: This is a bikeshed question, but any reason not to put this in the rbac-permissions-operator namespace?

Comment (lisa) [reply deploy/sre-prometheus/100-rbac-permission-operator.PrometheusRule.yaml]: I'm not sure.  Let me check with the monitoring folks to see if they can be re-homed.

Comment (lisa) [reply deploy/sre-prometheus/100-rbac-permission-operator.PrometheusRule.yaml]: https://github.com/openshift/rbac-permissions-operator/pull/16 sets up for the move, and 582f32e handles the move.

**Comment by lisa** [comment]
I'll be closing this in favour of another avenue to ensure the operator is _able_ to apply the configurations provided to it.

**Pull request closed without merging**